lib/src/command/outdated.dart

```diff
@@ -31,32 +31,40 @@ class OutdatedCommand extends PubCommand {
   String get docUrl => 'https://dart.dev/tools/pub/cmd/pub-outdated';
 
   OutdatedCommand() {
-    argParser.addFlag('color',
-        help: 'Whether to color the output. Defaults to color '
-            'when connected to a terminal, and no-color otherwise.');
-
     argParser.addFlag('json',
-        help: 'Outputs the results in a json formatted report',
+        help: 'Outputs the results in a json formatted\n'
+            'report.',
         negatable: false);
 
+    argParser.addFlag('color',
+        help: 'Whether to color the output.\n'
+            'Defaults to color when connected to a\n'
+            'terminal, and no-color otherwise.');
+
     argParser.addFlag('up-to-date',
         defaultsTo: false,
-        help: 'Include dependencies that are already at the latest version.');
+        help: 'Include dependencies that are already at the\n'
+            'latest version.');
 
-    argParser.addFlag('pre-releases',
+    argParser.addFlag('prereleases',
         defaultsTo: false,
-        help: 'Include pre-releases when reporting latest version.');
+        help: 'Include prereleases when reporting latest\n'
```

Comment:
Maybe shorten to "Include pre-releases in latest version."

Reply:
Done